test/e2e/node/pod_resize.go

```diff
@@ -1408,14 +1441,133 @@ func doPodResizeErrorTests() {
 	}
 }
 
-var _ = SIGDescribe("Pod InPlace Resize Container [Feature:InPlacePodVerticalScaling]", func() {
-	featureGatePostAlpha = true
-	if fs, found := utilfeature.DefaultFeatureGate.DeepCopy().GetAll()[features.InPlacePodVerticalScaling]; found {
-		if fs.PreRelease == featuregate.Alpha {
-			featureGatePostAlpha = false
+func doPodResizeSchedulerTests() {
+	f := framework.NewDefaultFramework("pod-resize-scheduler")
+
+	ginkgo.It("pod-resize-scheduler-testcase1", func() {
+		nodes, _ := f.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		nodeNum := len(nodes.Items)
+		if nodeNum > 1 {
+			framework.Logf("There are more than 1 node in the testing cluster, so skip the pod-in-queue-resize-test!")
+			return
 		}
-	}
+
+		ginkgo.By("Checking the node allocatable CPU resources!")
+		node := nodes.Items[0]
+		nodeAllocatableResources := node.Status.Allocatable
+		nodeAllocatableCPU := nodeAllocatableResources.Cpu()
+		nodeAllocatableCPUValue := nodeAllocatableCPU.MilliValue()
+		nodeAllocatableCPUStr := nodeAllocatableCPU.String()
+		framework.Logf("Node allocatable CPU is %s", nodeAllocatableCPUStr)
+		testPodRequest := nodeAllocatableCPUValue / 2
+		testPodRequestStr := strconv.FormatInt(testPodRequest, 10) + "m"
+		framework.Logf("Test Pod Request CPU is %s", testPodRequestStr)
+		testPodRequestAfterResize := testPodRequest / 2
+		testPodRequestAfterResizeStr := strconv.FormatInt(testPodRequestAfterResize, 10) + "m"
```

Comment:
repeated code can go into helper function.

Reply:
Please see staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go
I instead of creating string, you can use resource.NewMilliQuantity to allocate a Quantity struct, and use Quantity.String() to get the string values for TestCaseInfo below - that way we are using well tested APIs